src/frontend/next/src/components/Footer.tsx

```diff
@@ -1,4 +1,3 @@
-import { FC } from 'react';
 import { makeStyles, Theme } from '@material-ui/core/styles';
```

Comment:
Split these `Footer` changes into a separate PR, since they are unrelated to the goals of this fix.

Reply:
@humphd : finished sir